### PR TITLE
Module for a role assumable by a github web identity.

### PR DESCRIPTION
**Pull request by julialawrence**
Relevant issue: https://github.com/ministryofjustice/modernisation-platform/issues/2593

A module to create a role assumable by a github web identity to be used existing OIDC providers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
-# Modernisation Platform Terraform Module Template 
+# Modernisation Platform Terraform Module Template
```
Comment (davidkelliott) [inline README.md:1]: Title needs amending and the standards badge image and link below please!

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+resource "aws_iam_role_policy_attachment" "policy-arns" {
+  for_each   = toset(var.policy_arns)
+  role       = aws_iam_role.this.name
+  policy_arn = each.key
+}
+data "aws_iam_policy_document" "combined-role-policy" {
+  count                   = length(var.policy_jsons) > 0 ? 1 : 0
+  source_policy_documents = var.policy_jsons
```
Comment (davidkelliott) [inline main.tf:43]: Didn't realise you could do this, nice!

File: README.md (hunk 1 above)
Comment (julialawrence) [reply README.md]: Done!

File: test/module_test.go [hunk 3]
```diff
@@ -17,11 +18,14 @@ func TestModule(t *testing.T) {
 	defer terraform.Destroy(t, terraformOptions)
 
 	terraform.Init(t, terraformOptions)
+
 	terraform.WorkspaceSelectOrNew(t, terraformOptions, "testing-test")
```
Comment (davidkelliott) [inline test/module_test.go:22]: Do we want to remove this now if there is no backend here?

**Pull request merged**